client/assets/js/snake.js

```diff
@@ -19,8 +19,17 @@ export default class Snake {
 		this.height = SNAKEHEIGHT;
 
 		this.bodyParts = [];
+		this.name = name;
+	}
+	
+	getName(){
```

Comment:
The name can already be gotten from outside using `.name`. Please remove.
If stronger access control is needed, we'll use accessors (object getter and setters)